requirements.txt

```diff
@@ -1,5 +1,5 @@
-dj-database-url==0.4.1
 Django==1.10.4
+django-environ==0.4.1
 gunicorn==19.6.0
-psycopg2==2.6.2
+six==1.10.0
```

Comment:
This line is automatically written by pip-freeze
And if i'm not wrong - this is dependency for django-environ

`ImportError: No module named 'six'`